java/client/test/org/openqa/selenium/ElementAttributeTest.java

```diff
@@ -227,6 +228,14 @@ public void testShouldReturnTheContentsOfATextAreaAsItsValue() {
   }
 
   @Test
+  public void testShouldReturnInnerHtml() {
+    driver.get(pages.simpleTestPage);
+
+    String html = driver.findElement(By.id("wrappingtext")).getAttribute("innerHTML");
```

Comment:
this makes me wonder if we want to add a `getProperty()` in order to disambiguate properties from attributes..